README.md

```diff
@@ -80,6 +80,7 @@ People are writing great tools and papers for improving outputs from GPT. Here a
 - [Semantic Kernel](https://devblogs.microsoft.com/semantic-kernel/): A Python/C# library from Microsoft that supports prompt templating, function chaining, vectorized memory, and intelligent planning.
 - [Outlines](https://github.com/normal-computing/outlines): A Python library that provides a domain-specific language to simplify prompting and constrain generation.
 - [Promptify](https://github.com/promptslab/Promptify): A small Python library for using language models to perform NLP tasks.
+- [Humanloop](https://https://humanloop.com/): A paid product for building, evaluating and improving LLM apps.
```

Comment:
typo here, it should be `https://www.humanloop.com`

Reply:
@Razcle bumping this for you^